setup_env.sh

```diff
@@ -106,3 +106,16 @@ else
         fi
     fi
 fi
+
+
+# setting ECCODES_DEFINITION_PATH:
+git clone --recursive https://github.com/COSMO-ORG/eccodes-cosmo-resources.git
+
+${CONDA} activate ${DEV_ENV_NAME}
+conda_loc=${CONDA_PREFIX}
+base_dir=$(pwd)
+def_path_default=${conda_loc}/share/eccodes/definitions
+def_path_resources=${base_dir}/eccodes-cosmo-resources/definitions
+
+conda env config vars set ECCODES_DEFINITION_PATH=${def_path_default}:${def_path_resources}
```

Comment:
I think here it would be better to append your new paths to the `ECCODES_DEFINITION_PATH` instead of resetting it to avoid any issues when running ICON with the probtest environment activated. Do you think this could cause problems for your implementation?

Reply:
Ok, so I checked and at least for the no-ensemble part these two paths are fine. I have no idea why, but it seems we need both and as you said the default needs to be first for some reason. This looks very fishy to me, but okay for now, we will investigate this later.